src/App.js

```diff
@@ -0,0 +1,35 @@
+import {
+  BrowserRouter, Link, Route, Routes,
+} from 'react-router-dom';
+// eslint-disable-next-line
+import Books from "./components/Books";
+// eslint-disable-next-line
+import Categories from "./components/Categories";
+
```

Comment:
- [ ] The way you are importing the functions is incorrect. The correct paths to the files are:

`"./Components/Books"` and `"./Components/Categories'`. In your path for the import statement, you are using lowercase _**c**_ instead of uppercase _**C**_ for the `components` word. This causes the error I attached in the screenshot below


![image](https://user-images.githubusercontent.com/77106962/217838618-fc955ffa-6c5b-492b-90ef-75b641438ef4.png)

#### Kindly fix this error 👍 😎